pkg/collector/reader.go

```diff
@@ -251,7 +254,11 @@ func (c *Collector) reader() {
 					v.CurrCPUInstr = 0
 					v.CurrBytesRead = 0
 					v.CurrBytesWrite = 0
+<<<<<<< HEAD
```

Comment:
looks a merge conflict

Reply:
@rootfs sorry for that. just amend the update to solve the conflict.